### PR TITLE
feature #772: Created endpoint for receiving all cathedras

### LINKED ISSUE #772: Create endpoint for receiving all cathedras
As you have seen, we have the opportunity to create cathedras. However, we don't have an endpoint to get all cathedras for admin panel. So you have to create it, **requirements**:
- general query for cathedra entity
- pagination
- sorting (by name or abbreviation)
- order (asc | desc)
- response: cathedras with teachers

Also, don't forget about _Swagger documentation_

### PR DESCRIPTION
**Pull request by illiakroshka**
- Created DbCathedra
- Created findMany and count methods in repository
- Created getAll method in the service
- Created QueryAllCathedrasDTO
- Implemented sorting(name, abbreviation),order (asc, desc), search (name, abbreviation)
- Added Swagger documentation

closes #772

### CODE REVIEW

File: src/v2/database/repositories/CathedraRepository.ts [hunk 1]
```diff
@@ -25,4 +25,19 @@ export class CathedraRepository {
       include: this.include,
     });
   }
+
+  async findMany (
+    data: Prisma.CathedraFindManyArgs,
```
Comment (f-ogoog) [inline src/v2/database/repositories/CathedraRepository.ts:30]: needless enter

**Review by f-ogoog — CHANGES_REQUESTED** [review]

**Review by f-ogoog — APPROVED** [review]

**Review by Yakorrr — APPROVED** [review]

**Review by fokaaas — CHANGES_REQUESTED** [review]
Check qodana warning

File: src/v2/api/controllers/CathedraController.ts [hunk 2]
```diff
@@ -29,6 +33,31 @@ export class CathedraController {
...
+    @Query() query: QueryAllCathedrasDTO,
+  ) {
+    const cathedras = await this.cathedraService.getAll(query);
+    return {
+      cathedras: cathedras.data,
+      pagination: cathedras.pagination,
+    };
+  }
+
+  @Access(PERMISSION.CATHEDRAS_CREATE)
```
Comment (fokaaas) [inline src/v2/api/controllers/CathedraController.ts:60]: Add it into ApiEndoint

**Review by fokaaas — CHANGES_REQUESTED** [review]
Pls simply return the number of teachers instead of the objects

**Review by Dmutre — APPROVED** [review]
QA test is complited

**Review by fokaaas — APPROVED** [review]

**Pull request merged**